packages/dns-discovery/package.json

```diff
@@ -10,13 +10,6 @@
       "import": "./dist/index.js"
     }
   },
-  "typesVersions": {
-    "*": {
-      "*": [
-        "dist/*"
```

Comment:
Originally I found out that if changing it to `./dist/*.d.ts` will fix the problem as well - my assumption it is because typescript need to have specific path / pattern to point only to `.d.ts` files. Strangely I see this mistake in docs so not sure. 

But since `typesVersions` are not needed and `types` will be used otherwise I am just removing this section.

Reply:
I tried to test it in other scenarios and couldn't find any edge cases.

Reply:
I think the was introduced in these PRs:
- https://github.com/waku-org/js-waku/pull/892
- https://github.com/waku-org/js-waku/pull/1097

This shouldn't be a problem for current changes.